### PR TITLE
More create tests

### PR DESCRIPTION
**Pull request by RomanLangrehr**
Added some more create tests and improved existing create tests.

### CODE REVIEW

File: src/final2/subtests/CreateTest.java [hunk 1]
```diff
@@ -20,7 +22,120 @@
 	public CreateTest() {
 		setAllowedSystemExitStatus(SystemExitStatus.WITH_0);
 	}
-	
+
+	public void createNorthAnt() {
```
Comment (pcworld) [inline src/final2/subtests/CreateTest.java:30]: Missing `@Test`, this test is not actually run

Comment (RomanLangrehr) [inline src/final2/subtests/CreateTest.java:30]: Oh, thanks.
T'll fix this...

Am 2015-03-27 um 17:10 schrieb pcworld:

File: src/final2/subtests/CreateTest.java [hunk 2]
```diff
@@ -20,7 +22,120 @@
...
+				"0a00",
+				"Z000"
+		};
+		runs = new Run[] {
+				new ExactRun("ant", is("a,b,z")),
+				new NoOutputRun("create H,1,3"),
+				new ExactRun("ant", is("a,b,h,z")),
+				new ExactRun("position h", is("1,3")),
+				new ExactRun("direction h", is("N")),
+				new ExactRun("field 1,3", is("b")),
```
Comment (pcworld) [inline src/final2/subtests/CreateTest.java]: Shouldn't this be `is("h")`?

Comment (RomanLangrehr) [inline src/final2/subtests/CreateTest.java]: Alright.

Am 2015-03-27 um 17:14 schrieb pcworld:

**Comment by pcworld** [comment]
lgtm

**Comment by RomanLangrehr** [comment]
By the way: 
Fixes #139 
Fixes #159

**Pull request merged**